custom/icds_reports/tasks.py

```diff
@@ -704,7 +696,8 @@ def email_dashboad_team(aggregation_date, aggregation_start_time, force_citus=Fa
     icds_data_validation.delay(aggregation_date)
 
 
-@periodic_task(
+@periodic_task_on_envs(
+    settings.ICDS_TESTING_ENVS,
```

Comment:
This name is misleading. I thought it was only the 'testing' envs but it's actually all ICDS envs. Maybe we should make it 'ALL_ICDS_ENVS` and `PRIMARY_ICDS_ENVS`?

Also we can remove `icds-new`

Reply:
I think we can just use `ICDS_ENVS`. I don't htink that the stagnant cases recalculation is important on india
```suggestion
    settings.ICDS_ENVS,
```

Reply:
OK I'll take this suggestion and get rid of `ICDS_TESTING_ENVS`

Reply:
Point taken on the naming though. If I do end up needing to include india on one of these in the future, I'll use something like what you suggested, Simon